lib/graphql/query/literal_input.rb

```diff
@@ -21,7 +21,7 @@ def self.from_arguments(ast_arguments, argument_defns, variables)
             arg_value = coerce(arg_defn.type, ast_arg.value, variables)
           end
           if arg_value.nil?
-            arg_value = arg_defn.default_value
+            arg_value = arg_defn.type.coerce_input(arg_defn.default_value)
```

Comment:
No; `arg_defn.default_value` is not an AST node instance so calling `LiteralInput.coerce` on it will crash. That's why I'm calling `arg_defn.type.coerce_input` directly here.